### PR TITLE
Add class that can invite a user to a given bitwarden org

### PR DESCRIPTION
**Pull request by cob16**
The invited user has:
- Their “type” set to 2 (i.e a regular User)
- resetPasswordEnrolled enabled
- Their “externalId” set to the same as the username they were invited with

### CODE REVIEW

File: bitwarden_manager/bitwarden_public_api.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        self.__client_secret = client_secret
+        self.__client_id = client_id
+
+    def invite_user(self, username, email):
+        bearer = self.__fetch_token()
+        response = post(
+            f"{API_URL}/members",
+            headers={"Authorization": f"Bearer {bearer}"},
+            json={
+                "type": 2,
```
Comment (baarkerlounger) [inline bitwarden_manager/bitwarden_public_api.py:18]: Think it'd be worth extracting 2 out into a named constant or enum

Comment (cob16) [reply bitwarden_manager/bitwarden_public_api.py:18]: good idea :)

File: bandit.yaml [hunk 2]
```diff
@@ -1,2 +1,3 @@
 assert_used:
   skips: ['*_test.py', '*test_*.py']
+exclude_dirs: ["tests"]
```
Comment (baarkerlounger) [inline bandit.yaml:3]: If we're excluding tests altogether we don't really need to skip checking for use of asserts in tests above

File: Makefile [hunk 3]
```diff
@@ -16,12 +16,15 @@ install-poetry:
...
+	poetry install
 	poetry run pre-commit autoupdate
 
 static-analysis:
-	poetry run bandit -c bandit.yaml -r .
+	poetry run bandit -c bandit.yaml -r -q .
 	poetry run flake8 . *.py
 
-test:
+python-test:
```
Comment (baarkerlounger) [inline Makefile:26]: If you change this you'll also need to change this https://github.com/hmrc/build-jobs/blob/main/jobs/live/platsec.groovy#L215

Might be easier to change 29

Comment (cob16) [reply Makefile:26]: I want it to be realy easy to find these problems locally before the pr builder fails hence building it into after a test run with `make test` I will make the other pr if that's ok?

**Pull request merged**